lib/cfg.c

```diff
@@ -346,6 +346,8 @@ cfg_new_snippet(gint version)
   stats_options_defaults(&self->stats_options);
 
   cfg_tree_init_instance(&self->tree, self);
+  plugin_context_init_instance(&self->plugin_context);
```

Comment:
I think this init is not necessary here as g_new0 will zero it out by default.

Reply:
I see this init is extended with other things than memset in a later commit. My comment was intended only on the memset part.